packages/orbit-components/src/InputSelect/InputSelect.stories.tsx

```diff
@@ -1,267 +1,225 @@
+  },
+  {
+    title: "Squirtle",
+    value: "Squirtle",
+    group: "Starters",
+    description:
+      "Squirtle's shell is not merely used for protection. The shell's rounded shape and the grooves on its surface help minimize resistance in water, enabling this Pokémon to swim at high speeds.",
+  },
+  {
+    title: "Jigglypuff",
+    value: "Jigglypuff",
+    group: "Others",
+    description:
+      "Jigglypuff's vocal cords can freely adjust the wavelength of its voice. This Pokémon uses this ability to sing at precisely the right wavelength to make its foes most drowsy.",
+  },
+  {
+    title: "Gengar",
+    value: "Gengar",
+    group: "Evolutions",
+    description:
+      "Gengar is a shadow-like Pokémon that lurks in the darkness. It is said to emerge from darkness to steal the lives of those who become lost in mountains.",
+  },
+  {
+    title: "Dragonite",
+    value: "Dragonite",
+    group: "Evolutions",
+    description:
+      "Dragonite is capable of circling the globe in just 16 hours. It is a kindhearted Pokémon that leads lost and foundering ships in a storm to the safety of land.",
+  },
+  {
+    title: "Mewtwo",
+    value: "Mewtwo",
+    description:
+      "Mewtwo is a Pokémon that was created by genetic manipulation. However, even though the scientific power of humans created this Pokémon's body, they failed to endow Mewtwo with a compassionate heart.",
+  },
+  {
+    title: "Gyarados",
+    value: "Gyarados",
+    group: "Evolutions",
+    description:
+      "Gyarados is a Pokémon that has been known to cause major disasters. A vicious Pokémon from the sea, it appears wherever there is conflict to incite rage and cause destruction.",
+  },
+  {
+    title: "Eevee",
+    value: "Eevee",
+    group: "Starters",
+    description:
+      "Eevee has an unstable genetic makeup that suddenly mutates due to the environment in which it lives. Radiation from various stones causes this Pokémon to evolve.",
+  },
+];
+
+const meta: Meta<typeof InputSelect> = {
   title: "InputSelect",
+  component: InputSelect,
+
+  parameters: {
+    controls: {
+      exclude: ["onChange", "onFocus", "onClose", "onOptionSelect", "onBlur", "onSelect"],
+    },
+  },
+
+  args: {
+    options,
+    onChange: action("onChange"),
+    onFocus: action("onFocus"),
+    onOptionSelect: action("onOptionSelect"),
+    onClose: action("onClose"),
+    onBlur: action("onBlur"),
+    onSelect: action("onSelect"),
+    placeholder: "Select currency",
+    prevSelectedLabel: "Previously selected",
+    showAll: true,
+    maxHeight: "400px",
+    readOnly: false,
+    emptyState: "No results found.",
+    labelClose: "Close",
+  },
 };
 
-export const Grouped = ({ options, showAll, readOnly }) => {
-  return (
+export default meta;
+type Story = StoryObj<typeof InputSelect>;
+
+export const Grouped: Story = {
+  render: args => (
     <div className="min-h-[1000px]">
```

Comment:
Is it needed to have the `InputSelect` wrapped by `div` element?

I think it can be rewritten to:
```
export const Grouped: Story = {
  parameters: {
  ...
}
```

but maybe I haven't caught anything. 🤔 :D

Reply:
Yes, it's there so when you open the select, it doesn't open over the source code. I think it makes sense to have it there but I guess 1000px is a bit of an overkill. Since `maxHeight` is set to 400px, I changed this min-h to 500px.